rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import babel from "@rollup/plugin-babel";
+import commonjs from "@rollup/plugin-commonjs";
+import json from "@rollup/plugin-json";
+import resolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import svgr from "@svgr/rollup";
+import analyze from "rollup-plugin-analyzer";
+import cleaner from "rollup-plugin-cleaner";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+
+const plugins = [
+  cleaner({
+    targets: ["./dist/"],
+  }),
+  peerDepsExternal(),
+  replace({
+    "process.env.NODE_ENV": JSON.stringify("production"),
```

Comment:
Are you using `process.env.NODE_ENV` anywhere in this repo?
If yes, why is it hardcoded as `production`?

Reply:
No. This is the only place where `process.env.NODE_ENV` is used.